httpstan/cache.py

```diff
@@ -18,6 +18,11 @@
 logger = logging.getLogger("httpstan")
 
 
+def cache_filename() -> str:
+    cache_path = appdirs.user_cache_dir("httpstan", version=httpstan.__version__)
```

Comment:
This is probably ok, but should this get \_\_version\_\_ from relative location?

```suggestion
    cache_path = appdirs.user_cache_dir("httpstan", version=.__version__)
```

Reply:
I think the verbose version is easier to read.